cdap-ui/app/cdap/components/EntityCard/FastActions/index.js

```diff
@@ -49,6 +49,11 @@ export default class FastActions extends Component {
   onSuccess(action) {
     if (action === 'startStop') { return; }
 
+    if (action === 'setPreferences') {
+      this.props.onPreferencesSaved();
```

Comment:
As mentioned above this should have been `this.props.onSuccess()`. Preference as it is a FastAction. We should not have anything outside the scope of `FastAction` here as it is going to be the base for all FastActions.